src/components/Summary.js

```diff
@@ -277,33 +304,45 @@ export default class Summary extends Component {
          </div>
       );
     });
+    return (<div>
+      {subSections}
+      <DataInfo
+            contentText={summaryMap[section].provenanceText}
+            queryDateTime={formatit.currentDateTimeFormat()}
+          />
+    </div>);
   }
 
   renderSectionHeader(section) {
     const flagged = this.isSectionFlagged(section);
     const flaggedClass = flagged ? 'flagged' : '';
-    const { numMedicalHistoryEntries, numPainEntries, numTreatmentsEntries, numRiskEntries, numNonPharTreatmentEntries, numExternalDataEntries } = this.props;
+    const { numMedicalHistoryEntries, numPainEntries, numTreatmentsEntries, numRiskEntries, numNonPharTreatmentEntries, numPDMPDataEntries } = this.props;
 
     let icon = '';
-    let title = '';
+    let sourceTitle = summaryMap[section]['title'];
+    let title = sourceTitle;
     if (section === 'PertinentMedicalHistory') {
-      icon = <MedicalHistoryIcon width="30" height="40" />;
-      title = `Pertinent Medical History (${numMedicalHistoryEntries})`;
+      icon = <MedicalHistoryIcon width="25" height="35" />;
+      title += ` (${numMedicalHistoryEntries})`;
     } else if (section === 'PainAssessments') {
       icon = <PainIcon width="35"  height="35" />;
-      title = `Pain Assessments (${numPainEntries})`
+      title += ` (${numPainEntries})`
     } else if (section === 'HistoricalTreatments') {
-      icon = <TreatmentsIcon width="36" height="38" />;
-      title = `EHR Medications (${numTreatmentsEntries})`
+      icon = <MedicineIcon className={`sectionIcon`} title={`${sourceTitle}`} />;
+      title += ` (${numTreatmentsEntries})`
     } else if (section === 'RiskConsiderations') {
       icon = <RiskIcon width="35" height="34" />;
-      title = `Risk Considerations (${numRiskEntries})`;
-    } else if (section === 'ExternalDataSet') {
-      icon = <MedicalHistoryIcon width="30" height="40" />;
-      title = `State PMP Prescriptions (${numExternalDataEntries})`;
+      title += ` (${numRiskEntries})`;
+    } else if (section === 'PDMPMedications') {
+      icon = <RxIcon width="20" height="25" className={`sectionIcon`} title={`${sourceTitle}`}/>;
+      title += ` (${numPDMPDataEntries})`;
     } else if (section === 'NonPharmacologicTreatments') {
       icon =  <TreatmentsIcon width="36" height="38" />;
-      title = `Non-Pharmacologic Treatments (${numNonPharTreatmentEntries})`;
+      title += ` (${numNonPharTreatmentEntries})`;
+    } else if (section === 'Occupation') {
+      icon = <OccupationIcon className={`sectionIcon`} title={`${sourceTitle}`}/>;
+    } else if (section === 'PatientEducationMaterials') {
+      icon = <UserIcon className={`sectionIcon`} title={`${sourceTitle}`} />;
```

Comment:
Surprised this wasn't implemented as an object that maps section names to titles...